heclib/Dss7/Public/zinquireChar.c

```diff
@@ -38,7 +38,7 @@ int zinquireChar(long long *ifltab, const char *request, char *creturn, size_t c
 		}
 		else {
 			len = stringLastNonBlank(creturn, creturnSize);
-			creturn[len] = '\0';
+			creturn[len + 1] = '\0';
```

Comment:
I think this change needs to be a separate branch until we can also test in Solaris/big-endian.